aiidalab_widgets_base/structures.py

```diff
@@ -119,7 +119,8 @@ def _structure_importers(self, importers):
         # If there is only one importer - no need to make tabs.
         if len(importers) == 1:
             # Assigning a function which will be called when importer provides a structure.
-            importers[0].connect_to_manager(self)
+            if importers[0].has_trait('structure'):
+                dlink((importers[0], 'structure'), (self, 'input_structure'))
```

Comment:
I think there is an expectation that *every* importer and editor *must have* the `.structure` trait, correct? We should only guard against potentially not existing trait that are explicitly optional.